### DISCUSSION TITLE
How to make DCEF show deleted messages?

### DISCUSSION THREAD

**Discussion by WanHaes** [slatinsky/DiscordChatExporter-frontend#50] [Q&A]
This is unclear. If I put two json files, one with deleted messages and one without them, DCEF shows deleted messages as if they were not deleted (without red highlight). I tried to clear mongodb cache, it doesn't help.

**Comment by slatinsky** [comment]
Hello, deleted messages will be detected only in the intersection of two channel exports.

**Reply by WanHaes** [reply]
Hi, yes, I tried to make two channel exports, with and without some messages, renamed one of them and put them together in exports.
